src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,39 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. It's a good practice to make fields private and provide public getters and setters to access them, which you've done, but the fields themselves should also be declared as private to prevent direct access.